apisix/cli/ngx_tpl.lua

```diff
@@ -550,6 +554,23 @@ http {
     }
     {% end %}
 
+    {% if status_standalone then %}
+    server {
+        listen {* status_server_addr *} enable_process=privileged_agent;
+        access_log off;
+        location /status {
+            content_by_lua_block {
+                apisix.status_standalone()
+            }
+        }
+        location /status/ready {
+            content_by_lua_block {
+                apisix.status_standalone_ready()
```

Comment:
We should provide a unified `/status/ready` endpoint that not only supports standalone mode but also does not support the mode based on etcd as the configuration center.